src/ipc/ipc-common/IPCProxyBase.h

```diff
@@ -49,10 +49,12 @@ class IPCProxyBase : public AdapterType, protected IPCProxyBaseBase
 
 public:
     using InterfaceType = AdapterType;
+    IsReadyObserver m_readyObserver{};
```

Comment:
Is this necessary to be public ?

Reply:
> I ask you reply all the comments!
> At least: "Done", "not needed", "to be done", or answer to the question.
> Otherwise I cannot see any reason for wasting the time and doing reviews when
> your comments picked selectively and yours questions are ignored!

Sorry me, but for save time, I try to make corrections without comment.

Reply:
no problem, otherwise it takes my time instead for finding older comments and compare with new version. Obvious cases you could mark as resolved as well.
I mean:
> change the variable name func -> functor
> Done
and mark as resolved or just only mark as resolved but the Done  comment will help.
I think we'll do review without squash in the future.